rl_algorithms/common/helper_functions.py

```diff
@@ -94,9 +94,11 @@ def numpy2floattensor(arrays: Tuple[np.ndarray]) -> Tuple[np.ndarray]:
     """Convert numpy arrays to torch float tensor."""
     tensors = []
     for array in arrays:
-        tensor = torch.FloatTensor(array).to(device)
-        if torch.cuda.is_available():
-            tensor = tensor.cuda(non_blocking=True)
-        tensors.append(tensor)
-
+        if isinstance(array, torch.Tensor):
```

Comment:
This method is that numpy array changes to float tensor. The function cannot take torch.tensor as a parameter.

Reply:
resolved in commit 0949a63.